x/programs/runtime/config.go

```diff
@@ -0,0 +1,178 @@
+// Copyright (C) 2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package runtime
+
+import "github.com/bytecodealliance/wasmtime-go/v12"
+
+const (
```

Comment:
should there be an SIMD default?

Reply:
default is disabled but it should be explicit I will set a default for every config option and try to document why.